NOnion/Utility/Base32Util.fs

```diff
@@ -163,4 +167,4 @@ module Base32Util =
                         tail
                 | _ -> result
 
-            [ for c in text -> c ] |> parse [] |> List.toArray
+            [ for c in text -> c ] |> parse List.Empty |> List.toArray
```

Comment:
@parhamsaremi this is a false negative for AvoidTooShortNames! Please file a bug about this tomorrow

Reply:
filed in https://github.com/fsprojects/FSharpLint/issues/592

Reply:
Ok thanks, you can try fixing it (add a test first). There's a chance that it's already fixed (maybe by this commit which is not contained on a publish release yet: https://github.com/fsprojects/FSharpLint/commit/c72dcd61785edc69f6cdbaee20bc40cefbdb1a8e ), but if it is, there's still value in adding the new unit test.

Reply:
great sure 👍